examples/gardener-controllerregistration.yaml

```diff
@@ -15,10 +15,12 @@ spec:
     type: google-clouddns
   - kind: DNSProvider
     type: openstack-designate
+  - kind: DNSProvider
+    type: cloudflare-dns
```

Comment:
Do we need both `examples/controller-registration.yaml` and `examples/gardener-controllerregistration.yaml`? Can we use only `examples/controller-registration.yaml`?